tests/test_endpoint.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/* Endpoint Tests */
+
+class Micropub_Endpoint_Test extends WP_UnitTestCase {
+
+
+	protected static $author_id;
+	protected static $subscriber_id;
+
+	// POST args
```

Comment:
can all this test data be reused from `test_micropub.php` instead of duplicated?

Reply:
I intended to remove the test_micropub file in favor of the endpoint file as part of the split. So it will be gone in a future commit.

Reply:
great! sounds good. mind adding a TODO?

Reply:
ping